src/contracts/atlas/GasAccounting.sol

```diff
@@ -163,7 +186,7 @@ abstract contract GasAccounting is SafetyLocks {
 
     function _trySolverLock(SolverOperation calldata solverOp) internal returns (bool valid) {
         if (_borrow(solverOp.value)) {
-            solver = solverOp.from;
+            _solverLock = uint256(uint160(solverOp.from));
```

Comment:
```suggestion
            _solverLock = uint256(uint160(solverOp.from)) | _solverVerified;
```
Should we include setting `_solverVerified` to `true` here?
Because `_trySolverLock` can only be reached if `verifySolverOp` results in `canExecute == true`

Reply:
ah - that's a different type of "verified."  In this case, I was referring to the "reconcile" callback. I'll change the name.